.github/workflows/test-reporter.yml

```diff
@@ -0,0 +1,19 @@
+---
+## Workflow to process the JUnit test results and add a report to the checks.
+name: Test Report
+
+on:
+  workflow_run:
+    workflows: [ci]
+    types: [completed]
+
+jobs:
+  report:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: dorny/test-reporter@c9b3d0e2bd2a4e96aaf424dbaa31c46b42318226
```

Comment:
to unify versions across repos may be worth to use a reusable workflow or wrap it in an action in the pam-pipeline-library repo

Reply:
```suggestion
    - uses: elastic/apm-pipeline-library/.github/actions/test-report@gh_actions
```